azure-pipelines.yml

```diff
@@ -45,18 +168,21 @@ stages:
     - bash: echo "##vso[task.setvariable variable=BUILD_NUMBER]$(expr $BUILD_NUMBER + 1)"
       condition: notIn(variables['BUILD_NUMBER'], variables['noSuch'])
       displayName: 'Iterate build number if it exists'
+
     # Set up environment and run the build
     - bash: conda install --yes --quiet -c conda-forge python=2.7 anaconda-client conda-build libiconv pathlib2
       displayName: 'Set up build environment'
-    - bash: conda build --python=2.7 -c conda-forge -c fermi -c fermi/label/root6 .
+
+    - bash: conda build --python=2.7 -c conda-forge -c fermi .
       displayName: 'Run the build'
-    - bash: anaconda -v -t $(Jasercion-Anaconda-Api) upload -u fermi --label=dev /usr/share/miniconda/conda-bld/linux-64/fermitools-*
+    - bash: anaconda -v -t $(Jasercion-Anaconda-Api) upload -u fermi --label=dev --label=$(TagName) --label=$(Build.SourceBranchName) --force /usr/share/miniconda/conda-bld/linux-64/fermitools-*
       displayName: 'Upload build to Anaconda Cloud'
 
-  ### Linux Test Stage ###
+
+  #### Linux Test Stage ###
 - stage: linux_test_stage
   displayName: 'Ubuntu Test Stage'
-  condition: succeeded('linux_build_stage')
+  dependsOn: linux_build_stage
```

Comment:
does `dependsOn` have an implicit success condition for the previous stage?

Reply:
yes